Frontend/android-native/mobility-driver/src/main/java/in/juspay/mobility/driver/MobilityDriverBridge.java

```diff
@@ -1095,6 +1120,106 @@ public boolean onRequestPermissionResult(int requestCode, String[] permissions,
         }
         return super.onRequestPermissionResult(requestCode, permissions, grantResults);
     }
+
```

Comment:
Make a separate class for all functions related to Camera/ Capture and move these functions in that class. Later create instance of the class and use those functions